Sources/Benchmark/BenchmarkExecutor.swift

```diff
@@ -154,7 +156,7 @@ struct BenchmarkExecutor { // swiftlint:disable:this type_body_length
             #endif
 
             if mallocStatsRequested {
-                startMallocStats = MallocStatsProducer.makeMallocStats()
+                MallocInterposerSwift.hook()
```

Comment:
You don't want to hook in this closure, it can be called twice as mentioned in the above comment - it is also unnecessary overhead to hook/unhook for each benchmark iteration, better to do the hooking outside of the loop - see how it's done for `ARCStatsProducer` which also hooks/unhooks, but does it outside the loop.